docs/custom-install.md

```diff
@@ -12,71 +12,70 @@ title: ''
+3. A custom Elemental client configuration file is, by default, located at `/etc/elemental/config.yaml`
+   and it can also be splitted into multiple yaml files under `/etc/elemental/config.d` directory.
+   See the configuration file
+   [reference](https://rancher.github.io/elemental-toolkit/docs/customizing/general_configuration/).
 
-Additional config files can be added dynamically boot time by generating the ISO via a [SeedImage](/seedimage-reference.md) including custom cloud-config data.
+A common pattern is to combine the three ways described above depending on the specific needs.
 
-To apply this pattern, the following files need to be included in the ISO or generated at boot time:
+### Adding additional cloud-config files within the installed OS
 
-1. A [configuration file](https://rancher.github.io/elemental-toolkit/docs/customizing/general_configuration/)
-   for the elemental client. The file must be named `config.yaml` and located by default in `/etc/elemental`.
-   This path can be configured as part of the installation parameters of a [MachineRegistration](/machineregistration-reference) resource.
+In order to include additional cloud-config files during the installation they need
+to be added to the installation data into the MachineRegistration resource. The most simple
+way to achieve it is by adding the desired cloud-config directly as part of the MachineRegistration [devoted
+section](machineregistration-reference.md#configcloud-config). See the example below
 
-2. The additional `cloud-init` files to be included into the installed system. They
-   allow to perform custom operations at boot time.
 
-3. The installation hooks are evalutated at install time. They allow to perform custom operations
-   during the installation process (include extra software, set additional disks...). The same
-   way `cloud-init` files can be included to perform custom operations during the live installation
-   media boot time.
-
-#### Custom Elemental client configuration file
-
-[Elemental client](https://github.com/rancher/elemental-toolkit/blob/main/docs/elemental.md) `install`, `upgrade` and `reset` commands can be configured with a [custom configuration file](https://rancher.github.io/elemental-toolkit/docs/customizing/general_configuration/) located by default in `/etc/elemental/config.yaml`.
-If you have multiple yaml files, you need to add them in the `/etc/elemental/config.d` directory.
-
-A simple example to set hooks location could be:
-
-```yaml
-extra-partitions:
-  - size: 10240
-    fs: ext4
-    label: EXTRA_PARTITION
+```yaml showLineNumbers
+apiVersion: elemental.cattle.io/v1beta1
+kind: MachineRegistration
+metadata:
+  name: my-nodes
+  namespace: fleet-default
+spec:
+  ...
+  config:
+    ...
+    cloud-config:
+      stages:
+        boot:
+        - name: "Setting root user password"
+          users:
+            root:
+              passwd: root
```

Comment:
`PermitRootLogin` is `no` by default in SL Micro 6.0.
This may be confusing for users.

Reply:
This is a SSH configuration only, the disabled root login applies only for remote logins. Probably we could use here another user or append the snippet for to allow SSH root connections.

Reply:
Ended up by using a different non 'root' user for the example.